test/controllers/callback_controller_test.rb

```diff
@@ -2,6 +2,10 @@
 
 require 'test_helper'
 
+class MockSessionStore < ActiveRecord::Base
```

Comment:
I'd move this to within `CallbackControllerTest` so that it's not polluting the global namespace.